src/App.js

```diff
@@ -1,23 +1,38 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import BooksList from './containers/BooksList';
+import BookForm from './containers/BookForm';
 
-function App() {
+
+function App({ books = [] }) {
```

Comment:
Opps! we miss some hard requirement:

You need to create `component` dir and move all presentational components inside it, that's; both `app.js` and `book.js`.
Components that connect to redux store are held in `src/container` while components that don't connect to redux store are held in `src/component` and they are called presentational component.

![image](https://user-images.githubusercontent.com/31897434/82297557-b944b800-99aa-11ea-99f5-46d26b277a86.png)

![image](https://user-images.githubusercontent.com/31897434/82297521-acc05f80-99aa-11ea-8185-f5d79f9a6c99.png)

Reply:
Solved!

The component is `/components` folder:

https://github.com/mcastorena0316/Bookstore_CMS/blob/b833bc1e94a0d626112aa06353e5f49fefbbf10b/src/components/App.js#L7